src/utils/rulesUtils.js

```diff
@@ -26,6 +37,29 @@ export function reorder(list, startIndex, endIndex) {
 	return result;
 }
 
+export function forceHighMidLowOrder(rules) {
+	const q_per_risk = {'Alto': 0, 'Medio': 0, 'Bajo': 0};
+	rules.forEach(rule => {
+		q_per_risk[rule.contagionRisk] += 1;
+	});
+	const current_index_per_risk = {'Alto': 0, 'Medio': q_per_risk['Alto'], 'Bajo': q_per_risk['Alto'] + q_per_risk['Medio']};
+	rules.forEach(rule => {
+		if (rule['contagionRisk'] === 'Alto') {
+			rule['index'] = current_index_per_risk['Alto'];
+			current_index_per_risk['Alto'] += 1;
+		}
+		if (rule['contagionRisk'] === 'Medio') {
+			rule['index'] = current_index_per_risk['Medio'];
+			current_index_per_risk['Medio'] += 1;
+		}
+		if (rule['contagionRisk'] === 'Bajo') {
+			rule['index'] = current_index_per_risk['Bajo'];
+			current_index_per_risk['Bajo'] += 1;
+		}
+	});
+	return rules.sort((r1, r2) => {return r1.index < r2.index ? -1 : 1});
```

Comment:
No se podria hacer un sorting por `contagionRisk` directamente? Y despues se hace un for, donde simplemente se le setea i al index de cada rule.

Reply:
Uh posta, ahí hice el cambio metiendo una función de cmp, quedó mucho mas lindo